Web/HTML/computer-code/intro-code.md

```diff
@@ -11,21 +11,48 @@ category: must-know
 
 stub: true
 
-
 tags:
   - deep
-
-
+  
+links:
+  - '[Code Sample](https://codepen.io/lizthedeveloper/pen/gzrZza){code}'
+  
+---
+# code - Display code in HTML
 ---
 ## Content
 
-New content to go here. The author must be updated to match a valid Enki account.
-
+In HTML the `<code>` element is used to style the content to display like a fragment of a computer code.
+Both the starting and ending tags are necessary.
+
+Example:
+```
+<p>
+  The function
+  <code>
+    calculateAverage() 
+  </code> 
+  calculates the average
+  of a set of numbers
+</p>
+```
+Result:
+ The funcion `calculateAverage()` calculates the average of a set of numbers.
+ 
+The default font style for the `<code>` element for most browsers is the monospace font.
+ 
+**Note: It's best to use the `<code>` element to style content for HTML elements, CSS rules, filenames and JS scripts.**
+ 
 ---
 ## Practice
 
 What content is best to style when using the `<code>` element?
 
+???
```

Comment:
Gotta change all these questions, where there are multiple true answers that could be improperly ordered